samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -332,16 +348,15 @@ std::string getTestInfoStreamHeader(benchmark_app::InputInfo& inputInfo) {
     return strOut.str();
 }
 
-std::map<std::string, std::vector<InferenceEngine::Blob::Ptr>> getBlobs(
-    std::map<std::string, std::vector<std::string>>& inputFiles,
-    std::vector<benchmark_app::InputsInfo>& app_inputs_info) {
-    std::map<std::string, std::vector<InferenceEngine::Blob::Ptr>> blobs;
+std::map<std::string, ov::runtime::TensorVector> getBlobs(std::map<std::string, std::vector<std::string>>& inputFiles,
```

Comment:
```suggestion
std::map<std::string, ov::runtime::TensorVector> getBlobs(const std::map<std::string, std::vector<std::string>>& inputFiles,
```

Reply:
changed to std::map<std::string, std::vector<std::string>>
because copy will be modified inside the function. Performance is not critical here.